### PR TITLE
Introduce new options to the command and shell modules: always_run, fact...

### PR DESCRIPTION
**Pull request by stoned**
..., ignore_changed and ignore_errors

### CODE REVIEW

**Comment by mpdehaan** [comment]
Glad to see this.

The "fact" stuff overlaps with register, so it probably should not be added.   I don't want to create too many overlapping ways of  doing things in Ansible so that could get a little confusing.

Similiarly, ignore_errors: True already exists as a generic task keyword.

I think "ignore_changed" should probably be done as a task keyword as well to be consistent.

We should probably also do the same (make it a language keyword) for check mode, so that we don't send the check mode flag down to the module either.

Basically these are all nice ideas, but I think I would like to see them made available to all modules rather than special cased for command.

**Comment by stoned** [comment]
OK understood, thanks.
I am all for a generic 'ignore_changed:' as well as a generic 'always_run:'.
But 'ignore_errors:' could use a kind of "quiet" option.
As for 'register:' a couple of options could be nice too, but I guess what I was thinking about could be provided
with  more jinja2 filters in core.
Closing this issue then.
